pages/reference/api-support-matrix.en-US.mdx

```diff
@@ -90,15 +90,16 @@ import { Callout } from 'nextra-theme-docs';
 | H264 Transcoding Profiles | Baseline, Main, High                                                                                                                                                                               |
 | Video players             | Supported tested: livepeer.js, video.js, hls.js, safari/ios native player, Theoplayer; Supported untested: JWPlayer, android native, Shaka, Exo, Bitmovin, Brightcove                              |
-| Video Delivery Location   | Supported tested: EU, USSupported untested: Asia, South America, Australia, Africa                                                                                                                 |
-| Live stream recording     | Only available for paying users. HLS playback URL (m3u8) available 5 minutes after session ends. Livepeer.com stores recording and serves playback via CDN. There is no limit on recording length. |
+| Video Delivery Location   | Supported tested: EU, US Supported untested: Asia, South America, Australia, Africa                                                                                                                 |
+| Live stream recording     | Only available for paying users. HLS playback URL (m3u8) is available 5 minutes after the session ends. Livepeer.studio stores recording and serves playback via CDN. The limit to the recording length is up to 12 hours max. |
 | Recording output format   | HLS playback URL (m3u8)                                                                                                                                                                            |
 | CDN delivery              | CDN delivery is only available for paying users.                                                                                                                                                   |
 | Non-CDN Delivery          | Non-CDN delivery for free tier users is limited to 10 viewers per account. Free tier users will have transcoding limited to 240p after 30 free minutes.                                            |
 | Custom CDN                | Only for custom pricing tier users.                                                                                                                                                                |
 | Stream Status             | active or inactive                                                                                                                                                                                 |
-| Stream length             | We can support streams up to 12 hours in length. Please reach out to contact@livepeer.org if you’re looking to run longer uninterrupted streams through Livepeer.com.                              |
+| Stream length             | We can support streams up to 28 days in length. Please reach out to contact@livepeer.org if you’re looking to run longer uninterrupted streams through Livepeer.studio.                              |
```

Comment:
`Livepeer.studio` -> Livepeer Studio

Reply:
brand guidelines are forthcoming but we should not use `livepeer.studio` unless referring to the website